Contentless/Program.cs

```diff
@@ -43,11 +44,64 @@ public static class Program {
         var excluded = config.ExcludedFiles.Select(Program.MakeFileRegex).ToArray();
         var overrides = Program.GetOverrides(config.Overrides).ToArray();
 
+        var referencesVersions = config.References.ToDictionary(x => x, x => (string)null, StringComparer.OrdinalIgnoreCase);
+        if (config.References.Length > 0)
+        {
+            if (args.Length < 2) {
+                Console.WriteLine("You supplied references but there is no project file, this isn't compatible. Please specify the full path of project file, if you want to sync references");
+                return;
```

Comment:
Again, I don't think this should return, just print the warning and continue doing all the other stuff that isn't related to adding references.

Reply:
Fixed